packages/nimble-components/src/select/index.ts

```diff
@@ -567,15 +567,38 @@ export class Select
      */
     public override keydownHandler(e: KeyboardEvent): BooleanOrVoid {
         const initialSelectedIndex = this.selectedIndex;
-        super.keydownHandler(e);
         const key = e.key;
         if (e.ctrlKey || e.shiftKey) {
             return true;
         }
 
+        if (key !== keyArrowDown) {
+            super.keydownHandler(e);
+        }
+
         let currentActiveIndex = this.openActiveIndex ?? this.selectedIndex;
         let commitValueThenClose = false;
         switch (key) {
+            case keyArrowDown: {
```

Comment:
didn't trace the code path but just to confirm, no weird issues with the other interactions to consider? up, home, end?

Reply:
Looks like `up` is [supposed](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/menu_role#down_arrow) to select the last item in the list, but we haven't been doing that to date. 

This change now selects the item above the hidden selected menu-item, if my hacking is correct…

https://github.com/user-attachments/assets/43f1bfd2-0466-4eb8-9dc6-09418621f2ba

Reply:
Yep, needed to handle `ArrowUp` as well. Home and End have no issues as they don't rely on the current selection for their behavior.